fiduceo_uncertainties.f90

```diff
@@ -1494,39 +1516,15 @@ SUBROUTINE radiance_uncertainties(i,outData,coefs1,coefs2,coefs3,FCDR,&
        IF( twelve_micron_there )THEN
           FCDR%uce5(j,i)=outdata%noise_cnts(6,i)
        ENDIF
-       IF( outdata%nsmoothBB3(i) .gt. 0 )THEN
```

Comment:
And this is where the correct statistics are meant to be calculated - need to be put back in

Reply:
I have cross-checked with the old version and have re-entered the correct statistics calculations:

-->
       IF( outdata%nsmoothBB3(i) .gt. 0 )THEN
          FCDR%ucict3(i)=outdata%noise_cnts(4,i)/SQRT(1.*outdata%nsmoothBB3(i))
       ELSE
          FCDR%ucict3(i)=NAN_R
       ENDIF
       IF( outdata%nsmoothBB4(i) .gt. 0 )THEN
          FCDR%ucict4(i)=outdata%noise_cnts(5,i)/SQRT(1.*outdata%nsmoothBB4(i))
       ELSE
          FCDR%ucict4(i)=NAN_R
       ENDIF
       IF( twelve_micron_there )THEN
          IF( outdata%nsmoothBB4(i) .gt. 0 )THEN
             FCDR%ucict5(i)=outdata%noise_cnts(6,i)/SQRT(1.*outdata%nsmoothBB5(i))
          ELSE
             FCDR%ucict5(i)=NAN_R
          ENDIF
       ENDIF
       IF( outdata%nsmoothBB3(i) .gt. 0 )THEN
          FCDR%ucs3(i)=outdata%noise_cnts(4,i)/SQRT(1.*outdata%nsmoothSp3(i))
       ELSE
          FCDR%ucs3(i)=NAN_R
       ENDIF
       IF( outdata%nsmoothBB3(i) .gt. 0 )THEN
          FCDR%ucs4(i)=outdata%noise_cnts(5,i)/SQRT(1.*outdata%nsmoothSp4(i))
       ELSE
          FCDR%ucs4(i)=NAN_R
       ENDIF
       IF( twelve_micron_there )THEN
          IF( outdata%nsmoothBB3(i) .gt. 0 )THEN
             FCDR%ucs5(i)=outdata%noise_cnts(6,i)/SQRT(1.*outdata%nsmoothSp5(i))
          ELSE
             FCDR%ucs5(i)=NAN_R
          ENDIF
       ENDIF